### ISSUE TITLE
Devices appears only with Read Permission in IOT setup

### ISSUE THREAD

**Issue by hotmtk** [jasonacox/tinytuya#454]
@ IOT wizard, in pop-up a "Link Tuya App Account" dialog, it is not appearing : chose "Automatic" and "Read, Write, and Manage". Therefore devices appears only with Read Permission

**Comment by hotmtk** [comment]
You are correct, even on site the devices have only Read Only Status, they can be commanded without any issues. The documentation (how-to tutorial) has to be re-written.

**Comment by jasonacox** [comment]
I made an update to the README. Did you happen to capture a screenshot?

**Comment by hotmtk** [comment]
Unfortunately I didn't.

**Comment by jasonacox** [comment]
Thank you for reporting this issue! This issue has been open for over 2 years with no recent activity. We are closing it to help keep the issue tracker focused and manageable.

If this problem is still present in the latest version of TinyTuya, please feel free to reopen this issue or open a new one with updated details (TinyTuya version, Python version, device type, and steps to reproduce). We'd love to help!

Thank you for your contribution to TinyTuya! 🙏
